.travis.yml

```diff
@@ -6,8 +6,9 @@ matrix:
       os: linux
       jdk: oraclejdk9
 
-    - name: macOS 10.13 JDK 10
+    - name: macOS
```

Comment:
can update the name accordingly. It seems to be using Mac OS X 10.13.6 JDK 11.01

Reply:
I want to wait for #556 first, before testing which approach to take. 🤐